deploy-manage/distributed-architecture/clusters-nodes-shards/node-roles.md

```diff
@@ -9,7 +11,6 @@ Any time that you start an instance of {{es}}, you are starting a *node*. A coll
 
 Each node performs one or more roles. Roles control the behavior of the node in the cluster.
 
-
 ## Set node roles [set-node-roles]
 
 You define a node’s roles by setting `node.roles` in [`elasticsearch.yml`](../../deploy/self-managed/configure-elasticsearch.md). If you set `node.roles`, the node is only assigned the roles you specify. If you don’t set `node.roles`, the node is assigned the following roles:
```

Comment:
> You define a node’s roles by setting `node.roles` in [`elasticsearch.yml`](../../deploy/self-managed/configure-elasticsearch.md)...

This page is currently flagged as applying to "stack" and no deployment types are indicated. But it seems to nonetheless be linking only to the self-managed context.  If this page *is* specific to self-managed, IMO it ought to have that added to the `applies_to` frontmatter.  If it's not supposed to be self-managed-specific then I think you might need to generalize some of this text (e.g. to not refer to the `elasticsearch.yml` file, since that's not how settings are managed in Elastic Cloud and perhaps linking to the other pages that talk about how to update settings in ECE / ECH too).

Reply:
marked as self for now. We will correct it afterwards